homeassistant/components/unifi/device_tracker.py

```diff
@@ -153,14 +154,17 @@ def entity_registry_enabled_default(self):
 
         Make sure to update self.is_wired if client is wireless, there is an issue when clients go offline that they get marked as wired.
         """
-        LOGGER.debug(
-            "Updating UniFi tracked client %s (%s)", self.entity_id, self.client.mac
-        )
         await self.controller.request_update()
 
         if self.is_wired and self.client.mac in self.controller.wireless_clients:
             self.is_wired = False
 
+        LOGGER.debug(
+            "Updating UniFi tracked client %s\n%s",
+            self.entity_id,
+            pformat(self.client.raw),
```

Comment:
this is ok for now, but you might want to check that it's logged before writing, as now you're formatting something that is most likely thrown out.  Example: https://github.com/NabuCasa/hass-nabucasa/blob/dev/hass_nabucasa/iot_base.py#L90-L91

Reply:
I will add that with a future.pr. Thanks